File: src/FSharp.Tests/CalculatorTests.fs
module FSharp.Tests.CalculatorTests

open Fixie.Integration
open Fixie.Assertions

let ``Should Add``() =
    let calculator = new Calculator()
    calculator.Add(2, 3).ShouldBe(5, (*In F#, we have to explicitly specify the expression parameter*)"calculator.Add(2, 3)")

let ``Should Subtract``() =
    let calculator = new Calculator()
    calculator.Subtract(5, 3).ShouldBe(2, (*In F#, we have to explicitly specify the expression parameter*)"calculator.Subtract(5, 3)")

File: src/TargetFramework.Tests/CalculatorTests.cs
namespace TargetFramework.Tests
{
    using Fixie.Integration;
    using Shouldly;

    class CalculatorTests
    {
        public void ShouldAdd()
        {
            var calculator = new Calculator();
            calculator.Add(2, 3).ShouldBe(5);
        }

        public void ShouldSubtract()
        {
            var calculator = new Calculator();
            calculator.Subtract(5, 3).ShouldBe(2);
        }
    }
}
